pwa/app/routes/apidocs.tsx

```diff
@@ -0,0 +1,308 @@
+          <p>
+            Before you get started using The Blue Alliance APIs, you need to be
+            familiar with a few elements of web technology. The Blue Alliance
+            APIs work by having your computer send a web request to our servers
+            asking for some piece of data, and our servers send the data back to
+            your computer. You can ask for information about teams or matches,
+            or even send us information, like letting us know there&apos;s a
+            robot photo we should add to our data set.
+          </p>
+          <p>
+            First, you need some way of sending HTTPS Requests to The Blue
+            Alliance&apos;s servers. This will be your primary means of
+            communication with TBA. For testing purposes, your web browser may
+            suffice. For more advanced applications, you may want to use an
+            external library, such as{' '}
+            <a href="https://www.w3schools.com/jquery/jquery_ajax_intro.asp">
+              jQuery
+            </a>{' '}
+            (Javascript),{' '}
+            <a href="http://docs.python-requests.org/en/master/">Requests</a>{' '}
+            (Python), or <a href="https://square.github.io/okhttp/">OkHttp</a>{' '}
+            (Java), or others.
+          </p>
+          <p>
+            You will also need to be familiar with{' '}
+            <a href="https://en.wikipedia.org/wiki/Json">JSON</a>, a
+            machine-readable format for sending and receiving data. Most of the
+            TBA APIs use JSON-formatted data, so you should find out how to
+            parse JSON text in the language of your choice.
+          </p>
+          <p>
+            Once you&apos;ve figured out how to make HTTPS requests, you will
+            need to figure out how to manipulate request and response headers.
+            These will be used to pass authentication keys to TBA and understand
+            the cache life of returned data.
+          </p>
+        </div>
+
+        <div>
+          <div>
+            <h3 className="text-2xl">
+              <Link to="/apidocs/v3">Read API (v3)</Link>
+            </h3>
+            <p>
+              Most people want to pull event listings, team information, match
+              results, or statistics from The Blue Alliance to use in their own
+              application. The read API is the way to do this. This API exposes
+              almost all of the data you see on TBA to your application in a
+              machine-readable format called JSON.
+            </p>
+          </div>
+
+          <div>
+            <h4 className="text-xl">API Endpoint</h4>
+            <p>
+              All requests should be made to the base URL:{' '}
+              <code>https://www.thebluealliance.com/api/v3</code>.
+            </p>
+          </div>
+
+          <div>
+            <h4 className="text-xl">Authentication</h4>
+            <p>
```

Comment:
This is missing a section. Can you check this with `src/backend/web/templates/apidocs_overview.html` and make sure everything is actually copied over properly?